app/controllers/collections_controller.rb

```diff
@@ -15,12 +15,12 @@ def edit
 
   def update
     @collection.update(collection_params)
-    redirect_to dashboard_path(current_user)
+    redirect_to profile_path(current_user)
```

Comment:
I would keep this as dashboard path but we can discuss! going to approve & merge everything now though